depthai_sdk/src/depthai_sdk/nn_models/deeplabv3/model.yml

```diff
@@ -3,11 +3,11 @@ description: >-
 documentation: https://github.com/PINTO0309/PINTO_model_zoo/blob/master/026_mobile-deeplabv3-plus/01_float32/download.sh
 task_type: semantic_segmentation
 files:
-  - name: FP16/deeplabv3p_person.xml
+  - name: FP16/deeplabv3.xml
```

Comment:
is this correct?

Reply:
Rollbacked, most likely changed it when used PyCharm refactoring, thanks